src/audio/drc/Testbench_drc_lin2dB/norm.c

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+#include "typdef.h"
+
+/* Count the left shift amount to normalize a 32 bit signed integer value
+ * without causing overflow. Input value 0 will result to 31.
+ */
+int norm_int32(int32_t val)
+{
+	int s;
+	int32_t n;
+
+	if (!val)
+		return 31;
+
+	if (val > 0) {
+		n = val << 1;
+		s = 0;
```

Comment:
```c
n = val << 1;
s = 0;
```
is done twice in both cases, this could be put once before the ```if (val > 0)```

Reply:
Yup, make sense - once is  enough